python/shotgun_desktop/errors.py

```diff
@@ -135,26 +135,27 @@ def __init__(self, reason, toolkit_path):
         )
 
 
-class ToolkitDisabledError(ShotgunDesktopError):
-    """
-    This exception notifies the catcher that Toolkit has not been enabled by the user on the site.
-    """
-
-    def __init__(self):
-        """Constructor"""
-        ShotgunDesktopError.__init__(
-            self,
-            "Toolkit has not been activated on your site. Please activate Toolkit before relaunching Shotgun Desktop.",
-        )
-
-
 class MissingPython3SupportError(ShotgunDesktopError):
     def __init__(self):
         """
         """
         super(MissingPython3SupportError, self).__init__(
             "The tk-desktop engine in your site configuration may not support Python 3.\n"
             "\n"
-            "You need to upgrade the tk-desktop engine to v2.5.3+ in your site configuration or "
+            "You need to upgrade the tk-desktop engine to v2.5.9+ in your site configuration or "
```

Comment:
No sense telling clients to upgrade to a version of desktop we know has bugs. Let's suggest the latest.